src/components/assets/AssetsContainer.js

```diff
@@ -1,30 +1,31 @@
-import React, {createClass} from 'react'
-import {getClient} from '../../services/contentfulClient'
+import React, { createClass } from 'react'
+import { getClient } from '../../services/contentfulClient'
 import Assets from './Assets'
 import Search from './Search'
 import styles from './AssetsContainer.css'
 import CSSModules from 'react-css-modules'
 import update from 'react-addons-update'
-const AssetsContainer = createClass({
 
-  getInitialState () {
+const AssetsContainer = createClass({
```

Comment:
I'm far away from being a react expert but I see different usages of `class` syntax and `createClass` by React.

Maybe we wanna unify these for consistency?

`class ToggleButton extends React.Component {`

vs.

`const AssetsContainer = createClass({`